pegasus/sites.v3/code.org/views/ai_carousel.haml

```diff
@@ -0,0 +1,35 @@
+.carousel-wrapper#ai-carousel
+  .carousel{style: "height: 165px"}
+    .slides
+      .slide.with-image
+        %img{src: "/images/ai/ai-quote-betia.png", alt: ""}
+        .text-wrapper
+          %p.quote-text=hoc_s(:ai_carousel_quote_02)
+          %p.no-margin-bottom Betia Bentley <span>•</span> CS Discoveries Teacher
+      .slide.with-image
+        %img{src: "/images/ai/ai-quote-seneca.png", alt: ""}
+        .text-wrapper
+          %p.quote-text=hoc_s(:ai_carousel_quote_01)
+          %p.no-margin-bottom Seneca Hart <span>•</span> CS Discoveries Teacher
+
+  .pagination
+    #ai-carousel-pagination
+
+%script{:src=>'/js/jquery.carouFredSel-6.2.1-packed.js'}
+%script{:src=>'/js/jquery.touchSwipe.min.js'}
+:javascript
+  $(document).ready(function() {
+
+    var id = '#ai-carousel';
+    // Define the carousel using the ID.
+    $(id + ' .slides').carouFredSel({
```

Comment:
This actually read on VoiceOver really nicely! But I wasn't able to tab navigate to the next slide. I'm not familiar with the `carouFredSel` plugin and a quick search didn't pull up an easy fix for this –– I don't think this is a blocker

Reply:
In the future I'd like to replace this plugin w/ either a native JS slider or a non-jQuery slider so we have more control over stuff like this, but this one is already used throughout the site.

Reply:
Got it, makes sense!